AutoCollection/ServerRequestParser.ts

```diff
@@ -110,6 +106,10 @@ class ServerRequestParser extends RequestParser {
         return this.requestId;
     }
 
+    public getCorrelationContext() {
+        return this.correlationContext;
```

Comment:
This is fairly confusing naming, given that this is not a `CorrelationContext` object